addons/website_event_track/models/event_event.py

```diff
@@ -8,31 +8,36 @@
 class EventType(models.Model):
     _inherit = 'event.type'
 
-    website_track = fields.Boolean('Tracks on Website')
-    website_track_proposal = fields.Boolean('Tracks Proposals on Website')
+    website_track = fields.Boolean(
+        string='Tracks on Website', compute='_compute_website_data',
+        readonly=False, store=True)
+    website_track_proposal = fields.Boolean(
+        string='Tracks Proposals on Website', compute='_compute_website_data',
+        readonly=False, store=True)
 
-    @api.onchange('website_menu')
-    def _onchange_website_menu(self):
-        if not self.website_menu:
-            self.website_track = False
-            self.website_track_proposal = False
+    @api.depends('website_menu')
+    def _compute_website_data(self):
+        for event_type in self:
+            if not event_type.website_menu:
```

Comment:
Maybe also assign a value if the fields are "None" as we'e seen it can break the Odoo.